students/10th/gonholee/westagram/posts/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class Post(models.Model):
+    user            =   models.ForeignKey('users.user',on_delete=models.CASCADE)
+    post_text       =   models.CharField(max_length=1500)
+    hearts          =   models.IntegerField(default=0)
+    image_url       =   models.URLField()
```

Comment:
max_length 지정해주는게 좋을것같습니다. 
```suggestion
    image_url       =   models.URLField(max_length=1000)
```